### PR TITLE
feat: only accept y/n on confirmation

### PR DESCRIPTION
**Pull request by hiro-o918**
This PR enhances the confirmByUser function by enforcing that only y or n inputs are accepted during the confirmation prompt. 
Additionally, it introduces support for canceling the operation using Ctrl+C.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -239,6 +243,52 @@ func createJobWithFileName(filename *string, job *batchv1.Job) error {
 	return createJob(f, job)
 }
 
+func confirmByUser(tty *tty.TTY) (bool, error) {
+	fmt.Fprint(tty.Output(), "Do you want to create a job with the change you just made? [y/N]\n")
```
Comment (kitagry) [inline main.go:247]:
```suggestion
	fmt.Fprint(tty.Output(), "Do you want to create a job with the change you just made? [y/n]\n")
```

File: main.go [hunk 2]
```diff
@@ -239,6 +243,52 @@ func createJobWithFileName(filename *string, job *batchv1.Job) error {
...
+			answerCh <- answer
+		}
+	}()
+
+	for {
+		select {
+		case sig := <-sigs:
+			if sig == syscall.SIGINT {
+				return false, nil
+			}
```
Comment (kitagry) [inline main.go:275]:
```suggestion
			return false, nil
```

**Review by kitagry — APPROVED** [review]
LGTM

**Comment by mski-iksm** [comment]
LGTM!

File: main.go (hunk 1 above)
Comment (hiro-o918) [reply main.go]: 9a5d9f8

File: main.go (hunk 2 above)
Comment (hiro-o918) [reply main.go]: 9a5d9f8

**Comment by kitagry** [comment]
Thank you!

**Pull request merged**